libs/build_api_v2.py

```diff
@@ -189,6 +189,7 @@ def build_region_summary(
     "vaccinationsCompletedRatio": CommonFields.VACCINATIONS_COMPLETED_PCT,
     "vaccinationsAdditionalDoseRatio": CommonFields.VACCINATIONS_ADDITIONAL_DOSE_PCT,
     "icuCapacityRatio": CommonFields.CURRENT_ICU,
+    "bedsWithCovidPatientsRatio": CommonFields.CURRENT_HOSPITALIZED,
```

Comment:
Do you understand exactly what this `METRICS_NAME_TO_COMMON_FIELD` is used for?  It's a bit confusing since there isn't a 1:1 mapping between metrics and common fields.  If you have useful context, would it be possible to add a comment to it?

Reply:
I don't really have a good sense, but I think it might be used to link metrics back to a data source in order to build the annotations.